### PR TITLE
Update procedure-types.ttl

### PR DESCRIPTION
**Pull request by nicorusti**
Good evening,
I'm conducting research about trasforming Italian public contracts data into linked Open Data, and I'm using your ontology. 
Italian laws include some special procedures for public contracts. They apply to a few public administrations, such as Parliament, Presidency of Republic, Supreme Court.  These procedures cannot be classified under any of the procedure types you specified. I would suggest to add a Special procedure type to make your schema more flexible. Anyone willing to use this procedure type, may be free to define a more precise concept by using skos:narrower . 

Thanks, 
Nicola Rustignoli.

### CODE REVIEW

**Comment by jindrichmynarz** [comment]
First, you can use any instance of `skos:Concept` as an object of `pc:procedureType` if you're OK with the following inference (based on [this restriction](https://github.com/opendatacz/public-contracts-ontology/blob/master/public-contracts.ttl#L181-L185)):

```
[] pc:procedureType :concept .
:concept skos:inScheme proctypes: .
```

You may also want to consider if your special procurement procedures cannot be covered by the [extended schema in the Public Procurement Ontology](http://contsem.unizar.es/def/sector-publico/pproc.html#ProcedureTypeScheme).

I'm not sure if a top concept `proctypes:Special` (or "Other" or "Miscellaneous") is a good way to solve extensibility. What do you think, @svatek?

**Comment by nicorusti** [comment]
Thank you for your help. Unfortunately, I could not find any suitable procedure type even in the Public Procurement Ontology. 

In order to let you better understand my difficulties in using the procedure types defined in the schemas, let me give you an insight of Italian law (Code of public contracts, legislative decree 163/2006, art. 17): 

> "The works, services and procurements for National Bank, armed and police forces, public companies in the fields of gas, electricity, water, transportation, post, oil and gas drilling, ports and airports, or when special security or secrecy measures are required or in order to preserve State security, according to law, can be executed in derogation of regulations concerning publicity of awarding procedures ... "

 In some cases, hence, the procedure type may be secret, or may be customized for some of those "special" entities. 

Furthermore, in order to preserve their  independence, supreme legislative, executive and judicial bodies (Parliament, Presidency of Republic and Executive, Supreme Court) are exempted, under some conditions, from the Code of public contracts. That means that they are allowed to freely define any contract award procedure by themselves, which, of course, cannot be classified under any of the procedure types you defined. 

All those organizations, even though they follow special regulations, must publish, by law, an xml file with contracts data according to a predefined xsd schema. The xsd schema, designed by the national control authority for public contracts, defines two “special” procedure types for the above mentioned cases, which cannot be categorized under any of the concepts in your scheme.

In conclusion, I still think it would be a good idea to make your schema more flexible, in order to add ability to represent also these exceptional procedure types, which, I suppose, may exist also in other European countries. 
Please, let me know what you think about it.

**Comment by svatek** [comment]
Dear Nicola,

Sorry for the delay in reacting. As you know, the current scheme contains 9 procedure types; afaik they have been derived from the TED scheme (Jindrich, correct me if I am wrong). Extending this schema is possible in principle, of course, since the PCO should not only support TED but also various national registries. However, it would make sense to make a survey first to see what the different national systems have as specifics that cannot be accomodated within the current scheme. 
I am not in favor of 'Special' as residual bin; rather I would prefer to clarify the semantics (incl. the legal background) of the procedure - which you already did to some degree - and then to name it less ambiguously.

We will make a discussion in the PCO creators' team soon, where we will also pay attention to your suggestion. Thanks a lot for your insights and interest in the ontology.

Vojtech Svatek
